### PR TITLE
Translations UI

### PR DESCRIPTION
**Pull request by Integral**
Improved and functional UI for translations inside metadata editor.

### CODE REVIEW

File: src/article/ArticleAPI.js [hunk 1]
```diff
@@ -410,23 +410,23 @@ export default class ArticleAPI {
...
-      subjectEl.parentNode.removeChild(subjectEl)
-      tx.delete(subjectEl.id)
-
-      const abstractEl = tx.createElement('trans-abstract').attr('xml:lang', languageCode).append(
-        tx.createElement('p')
-      )
-      const articleMeta = tx.find('article-meta')
-      articleMeta.append(abstractEl)
+      // HACK: attribute selector with colon is invalid
+      const titles = tx.findAll('trans-title-group')
```
Comment (oliver----) [inline src/article/ArticleAPI.js]: Interesting! Could you create an issue on substance? Thx :)

Comment (oliver----) [reply src/article/ArticleAPI.js]: Or is it an invalid css selector in general?

**Comment by Integral** [comment]
It is in general invalid

**Comment by oliver----** [comment]
@Integral just saw this pattern on some forum -- maybe this works (?): `[xml\:lang]`

**Comment by oliver----** [comment]
@Integral I have rebased your branch on the latest `metadata-editor`
Let's talk 'tomorrow'

**Pull request merged**